libsawtooth/src/client/rest.rs

```diff
@@ -101,6 +102,19 @@ impl SawtoothClient for RestApiSawtoothClient {
             |item: Result<Transaction, _>| item.map(|txn| txn.into()),
         )))
     }
+    /// List all blocks in the current blockchain
+    fn list_blocks(
+        &self,
+    ) -> Result<
+        Box<dyn Iterator<Item = Result<ClientBlock, SawtoothClientError>>>,
+        SawtoothClientError,
+    > {
+        let url = format!("{}/blocks", &self.url);
+
+        Ok(Box::new(
+            PagingIter::new(&url)?.map(|item: Result<Block, _>| item.map(|txn| txn.into())),
```

Comment:
Should be `block` instead of `txn`, I think

Reply:
Fixed